flake.nix

```diff
@@ -18,6 +18,13 @@
 
     plutip.url = "github:mlabs-haskell/plutip/8364c43ac6bc9ea140412af9a23c691adf67a18b";
     ogmios-datum-cache.url = "github:mlabs-haskell/ogmios-datum-cache/880a69a03fbfd06a4990ba8873f06907d4cd16a7";
+
+    # ogmios and ogmios-datum-cache nixos modules (remove and replace with the above after merging and updating)
+    ogmios-nixos.url = "github:mlabs-haskell/ogmios/marton/nixos-module";
+    ogmios-datum-cache-nixos.url = "github:mlabs-haskell/ogmios-datum-cache/marton/nixos-module";
+
+    cardano-node.url = "github:input-output-hk/cardano-node";
```

Comment:
`cardano-node` should be pinned to an explicit revision or ref. Currently we're on `v1.35.3`